### ISSUE TITLE
Using defaultRegion resets the country code on every focus

### ISSUE THREAD

**Issue by Zmetser** [MojtabaHs/iPhoneNumberField#53]
## Description

When using `defaultRegion` with a selectable country code flag, the country code resets back to the default every time the text field is focused.

It should keep the selected country code and not reset back to the default.

## Repro

`iOS 15.0.4`
`iPhoneNumberField 0.6.1`

1. Select a country code other than the default
2. Tap in the text field
3. => The country code resets back to the default

## Code

```
iPhoneNumberField(nil, text: $text)
                .defaultRegion(Locale.current.regionCode)
                .flagHidden(false)
                .flagSelectable(true)
```

**Comment by bereydev** [comment]
Same issue here

**Comment by Zmetser** [comment]
@bereydev No, I'm using it like this, and living with the issues it has and postponing having to deal with it.

```swift
iPhoneNumberField(nil, text: $phoneNumberFormatted, isEditing: $fieldFocus)
                .flagHidden(false)
                .flagSelectable(true)
                .prefixHidden(false)
                .autofillPrefix(true)
```

**Comment by MojtabaHs** [comment]
Fixed in v 0.10.0

**Comment by bereydev** [comment]
@MojtabaHs  First thank you for putting time an effort into making this library publicly available. We are a small team of iOS developers and really like this project. It could be really useful for lot of other developer if it was maintained more actively! I would like to help your and support your project by contributing to this if you are up to put us a main contributors to the project we could really make it ready for new version of iOS and even more versatile as it is already. Please feel free to check our website https://bereyziat.dev or contact me directly on bereyziat.developer@gmail.com we could have a little call as well.
